src/snap-confine.apparmor.in

```diff
@@ -94,19 +94,21 @@
 
     # for running snaps on classic
     mount options=(rw rslave) -> /,
-    /{tmp/snap.rootfs_*,}snap/ r,
-    /{tmp/snap.rootfs_*,}snap/** r,
+    /tmp/snap.rootfs_*/snap/ r,
+    /tmp/snap.rootfs_*/snap/** r,
```

Comment:
I'm not sure how this change is related to this PR, but the change is fine.

+1 on all changes.